ImgSmall.java

```diff
@@ -0,0 +1,65 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
+      resize(args);
+  }
+  public static void resize(String[] args) throws IOException{
+    if (args.length != 2){
+      System.out.println("please put in a url and a percent");
+      System.exit(0);
+    }
+    //get url object
+    URL url = null;
+    try{
+      url = new URL(args[0]);
+    } catch (MalformedURLException mfue){
+      System.out.println("check the url");
```

Comment:
Single space after url - otherwise the output will be squished e.g. 

"check the urlhttp://google.com"